### PR TITLE
Update deps and move to php 7.4

### PR DESCRIPTION
**Pull request by pyrech**
I also synchronized recipes and dropped sensio/framework-extra-bundle dependency

### CODE REVIEW

File: .env.test [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# define your env variables for the test env here
+KERNEL_CLASS='JoliCode\SecretSanta\Kernel'
+APP_SECRET='$ecretf0rt3st'
+SYMFONY_DEPRECATIONS_HELPER=999999
+PANTHER_APP_ENV=panther
```
Comment (damienalexandre) [inline .env.test:5]: This file does not seems necessary as we already specify viariables in phpunit.xml.dist? Or maybe we should switch them back here.

**Review by damienalexandre — APPROVED** [review]
LGTM.

File: .env.test (hunk 1 above)
Comment (pyrech) [reply .env.test:5]: It's generated by latest Symfony recipes, so I guess this is the current best practices. It feels more natural - at least to me - to edit env vars from a .env than editing .phpunit.xml so I guess we could probably clean the .phpunit.xml

Comment (pyrech) [reply .env.test:5]: The phpunit.xml.dist has indeed be cleaned in the official recipe : https://github.com/symfony/recipes/blob/master/symfony/phpunit-bridge/4.1/phpunit.xml.dist
So let's do it too :)

**Pull request merged**